tools/linter_lib/custom_check.py

```diff
@@ -589,6 +589,10 @@ def build_custom_checkers(by_lang):
          'description': "thick CSS attribute is under-specified, please use pixels.",
          'good_lines': ["border-width: 5px;"],
          'bad_lines': ["border-width: thick;", "border: thick solid black;"]},
+        {'pattern': r' #[0-9a-f]{3,6}(?![0-9a-z_-])',
```

Comment:
Maybe we should just use stylelint to do this. See https://github.com/adnrs96/zulip/commit/fb617959c51f66585530e0f002a5617c3b53e698

Reply:
I think we should actually remove this piece of code since stylelint does the job of checking colours to not be hex.